motor/providers/aws/provider.go

```diff
@@ -79,7 +81,7 @@ func New(pCfg *providers.Config, opts ...ProviderOption) (*Provider, error) {
 		return nil, errors.Wrap(err, "could not load aws configuration")
 	}
 	if cfg.Region == "" {
-		log.Warn().Msg("no AWS region found, using us-east-1")
+		log.Info().Msg("no AWS region found, using us-east-1")
```

Comment:
Found where?

Reply:
either `AWS_REGION` or `.aws/credentials`

Reply:
Can we make that part of the message? Otherwise it's not actionable by the user.

Reply:
I agree but would like not to do this as part of this PR. We need another round of changes for AWS to change the way we use credentials for AWS internally

Reply:
I think we need to get that PR as soon as possible. It's important for us to provide an actionable error message and right now this one isn't actionable. We already saw in Slack this week where users were confused around how these regions were set.